mwdb/web/src/commons/ui/Autocomplete.tsx

```diff
@@ -1,6 +1,20 @@
-import React, { useRef } from "react";
+import { useRef } from "react";
 
-export default function Autocomplete({
+type AutocompleteItem<T> = {
+    item: T;
+};
+
+type Props<T> = {
```

Comment:
Missing typing for `inputProps`
```suggestion
type Props<T> = React.HTMLProps<HTMLInputElement> & {
```